app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -219,6 +219,7 @@ def client_source
         assessor_last_name: :assessor_last_name,
         assessor_email: :assessor_email,
         assessor_phone: :assessor_phone,
+        match_group: :match_group,
```

Comment:
Does this need to be added to `cas_columns` as well? Not all the columns are repeated there, so I'm not sure what its for. https://github.com/greenriver/hmis-warehouse/blob/921dc27e5e0d1153d16964f691b3a46f7b9a7511/app/models/concerns/cas_client_data.rb#L26